content/en/tracing/trace_collection/otel_instrumentation/dotnet.md

```diff
@@ -15,14 +15,14 @@ further_reading:
 
 ## Requirements and limitations
 
-tktk .NET requirements
+- Datadog .NET tracing library `dd-trace-dotnet` version 2.21.0 or greater
 
 The following OTel features implemented in the Datadog library as noted:
 
 | Feature                               | Support notes                       |
 |---------------------------------------|-------------|-----------------------------|
-| [OTel Context propagation][1]         | [Datadog distributed header format][9] is used instead. | 
-| [Span processors][2]                  | Unsupported                                          | 
+| OTel Context propagation              | [Datadog distributed header format][9] is used instead. |
```

Comment:
Note: I removed the link for the OTel Context propagation because there is no section inside the OTel .NET docs that discuss context propagation